src/tasks/clone-blueprint.js

```diff
@@ -0,0 +1,32 @@
+import Task from '../models/task';
+import BlueprintCloner from '../models/blueprint-cloner';
+
+export default class extends Task {
```

Comment:
Lets add an explicit class name, if only to make our IDE's happier.

Reply:
FYI... none of the other Tasks currently specify a class name.

Reply:
well, that's true.  AirBNB recommend the existing way too: 
https://github.com/airbnb/javascript#modules--prefer-default-export

TIL